packages/react-aria-components/scripts/create-react-aria-components/templates/vanilla-css/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "create-react-aria-components-vanilla-css",
+  "version": "1.0.0",
+  "description": "A library of components built with React Aria Components. ",
+  "main": "index.js",
+  "modules": "dist/module.js",
+  "types": "dist/types/index.d.ts",
+  "source": "src/index.ts",
+  "exports": {
+    "types": "./dist/types.d.ts",
+    "import": "./dist/import.mjs",
+    "require": "./dist/main.js"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "parcel build",
+    "start": "storybook",
+    "storybook": "storybook dev -p 6006",
```

Comment:
Is this supposed to point to a `main.js` storybook config file? Getting an error that says it needs one (don't think one gets generated, would be nice if one were)

Reply:
Yeah, in this template I don't have storybook setup yet (I've been working on that separate from this PR). I was debating running `npx storybook init` for the user in the script, but I didn't like some of the default things (like the glob pattern would need to be updated). The other option would be for the templates to just have storybook setup already, but then they wouldn't stay up to date.